esmvalcore/preprocessor/_time.py

```diff
@@ -237,6 +245,14 @@ def monthly_statistics(cube, operator='mean'):
 
     operator = get_iris_analysis_operation(operator)
     cube = cube.aggregated_by(['month_number', 'year'], operator)
+
+    # Correct the time coordinate
+    cube.coord('time').points = cube.coord('time').units.date2num([
+        cell.point.replace(day=15, hour=12, minute=0, second=0, microsecond=0)
+        for cell in cube.coord('time').cells()])
+
+    cube.coord('time').bounds = None
+    cube.coord('time').guess_bounds()
     return cube
```

Comment:
do we really need `second` and `microsecond`?

Reply:
Thanks, I removed them. The iris function does not change the `second` and `microsecond`.